CHANGELOG.asciidoc

```diff
@@ -33,6 +33,8 @@ https://github.com/elastic/beats/compare/v6.5.3...6.5[Check the HEAD diff]
 
 *Affecting all Beats*
 
+- Update Golang to 1.10.6. {pull}9563[9563]
```

Comment:
do you normally change this to the backport pr number or leave it with the original?

Reply:
We leave to the original, its also referer in the backport commit message.